Description: urlparse: AttributeError: 'module' object has no attribute 'uses_fragment'
Author: Johannes Graumann <johannes_graumann@web.de>
Bug-Debian: http://bugs.debian.org/679067
Forwarded: no

diff --git a/yum/packages.py b/yum/packages.py
index 5ef9951..3d81745 100644
--- a/yum/packages.py
+++ b/yum/packages.py
@@ -39,7 +39,12 @@ from constants import *
 from operator import itemgetter
 
 import urlparse
-urlparse.uses_fragment.append("media")
+
+try:
+    urlparse.uses_fragment.append("media")
+except AttributeError, err:
+    print err
+
 from urlgrabber.grabber import URLGrabber, URLGrabError
 
 try:
diff --git a/yum/yumRepo.py b/yum/yumRepo.py
index e5e9ece..3e6d78a 100644
--- a/yum/yumRepo.py
+++ b/yum/yumRepo.py
@@ -19,7 +19,11 @@ import re
 import time
 import types
 import urlparse
-urlparse.uses_fragment.append("media")
+
+try:
+    urlparse.uses_fragment.append("media")
+except AttributeError, err:
+    print err
 
 import Errors
 from urlgrabber.grabber import URLGrabber
